helpers/shells/recreateRepoConfigFile.sh

```diff
@@ -17,24 +17,66 @@
 
 bwDataDir="/var/borgwarehouse"
 directoriesList=$(ls -A $bwDataDir)
+_AUTOSIZE=0
```

Comment:
Good improvement, thank you ! Can you please comment at the top of the script to indicate that there is this `-a` option and what it does ? 
That way, users who don't necessarily know bash will be able to use it :). Thanks